setup.py

```diff
@@ -82,8 +82,10 @@ def is_requirement(line):
 
 setup(
     name='concept-xblock',
-    version='0.3.0',
+    version='0.3.2',
     description='concept XBlock',   # TODO: write a better description.
+    long_description='This is a simple XBlock which will allows one to tag problems with concepts.',
```

Comment:
We can instead just add `readme.rst` content to `long_description`, ref: https://github.com/openedx/edx-ora2/blob/eb9e2ac68b254f11df1da74908c21d62cc5850b0/setup.py#L62-L63

Reply:
We don't have the README and version changes in this repo yet so I opted for simple approach.